assets/js/translator.js

```diff
@@ -335,6 +336,8 @@ if(modeEnabled('translation')) {
                 clearTimeout(timer);
             }
 
+            translatedAlready = false;
```

Comment:
Is it possible to use `translateRequest` instead?

Reply:
@sushain97 Doesn't `translateRequest` go to null after translating?

Reply:
Ah, good point. I forgot what the original ticket was for. How about clearing the timeout once a translation request completes?